### DISCUSSION TITLE
Localisation tag should be renamed to Localization

### DISCUSSION THREAD

**Discussion by nschonni** [mdn/mdn-community#166] [Localization] [answered]
Although the current is better to me, it doesn't match the writing guides in the org that say to use the US spelling for things. EX: behavior vs behaviour

**Comment by SphinxKnight** [comment]
@schalkneethling who can edit tags for the discussions?

**Comment by schalkneethling** [comment, marked as answer]
Hey @nschonni and @SphinxKnight, I have updated the category name.
